src/main/resources/intentionDescriptions/FlipBinaryExpressionIntention/description.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention flips the operator of a binary expression.
```

Comment:
Looks like the description for similar java intention is slightly better (see https://github.com/JetBrains/intellij-community/blob/master/plugins/IntentionPowerPak/src/intentionDescriptions/FlipExpressionIntention/description.html). Maybe use the same description?